.github/workflows/cypress-e2e-reporting-test.yml

```diff
@@ -150,7 +150,7 @@ jobs:
 
       - name: Run Cypress tests
         run: |
-          yarn cypress:run --browser chrome --headless --spec '.cypress/integration/*'
+          yarn cypress:run --browser chrome --headless
```

Comment:
Oops good catch on this, seems like we don't need to specify the `spec` for this directory setup. Thanks!